doc/source/index.rst.template

```diff
@@ -51,6 +51,7 @@ programming language.
                     <h5 class="card-title">User guide</h5>
                     <p class="card-text">The user guide provides in-depth information on the
                     key concepts of pandas with useful background information and explanation.</p>
+                </div>
```

Comment:
not sure why this `div` and `img` not originally indented as children of the outer `div`, same as below.